packages/TBC20/package.json

```diff
@@ -5,25 +5,25 @@
   "private": true,
   "type": "module",
   "devDependencies": {
-    "@types/bitcoin-computer__lib": "^0.17.1",
-    "@types/chai": "^4.3.1",
+    "@types/bitcoin-computer__lib": "^0.17.2",
```

Comment:
Same here, we should remove this devDependency to see if this change effectively work